packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -16,31 +15,47 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
           display: 'flex',
           alignItems: tokens.alignItems,
           flexDirection: tokens.flexDirection,
-          paddingHorizontal: globalTokens.size40,
+          paddingHorizontal: tokens.rootHorizontalPadding,
+          paddingVertical: tokens.rootVerticalPadding,
           ...borderStyles.from(tokens, theme),
         },
+        android_ripple: { color: tokens.rippleColor },
       }),
-      ['flexDirection', 'alignItems', ...borderStyles.keys],
+      ['flexDirection', 'rootHorizontalPadding', 'rootVerticalPadding', 'rippleColor', 'alignItems', ...borderStyles.keys],
     ),
     button: buildProps(
       (tokens: RadioTokens) => ({
         style: {
           backgroundColor: 'transparent',
-          width: tokens.radioSize,
-          height: tokens.radioSize,
+          width: tokens.radioOuterCircleSize,
```

Comment:
Shouldn't these be radioViewSize? The padding would make the outer circle smaller than radioOuterCircleSize.

Reply:
Thanks for catching this, realized margin will suite better, going with that instead